components/menu/package.json

```diff
@@ -14,27 +14,27 @@
     "url": "https://github.com/adobe/spectrum-css/issues"
   },
   "main": "dist/index.css",
-  "scripts": {
-    "build": "gulp",
-    "clean": "rimraf dist"
-  },
   "peerDependencies": {
-    "@spectrum-css/checkbox": ">=7",
+    "@spectrum-css/checkbox": ">=8",
     "@spectrum-css/divider": ">=2",
     "@spectrum-css/icon": ">=4",
     "@spectrum-css/switch": ">=4",
     "@spectrum-css/tokens": ">=13",
     "@spectrum-css/tray": ">=2.1"
   },
-  "devDependencies": {
-    "@spectrum-css/checkbox": "^8.0.3",
-    "@spectrum-css/component-builder-simple": "^3.0.1",
-    "@spectrum-css/divider": "^2.1.34",
-    "@spectrum-css/icon": "^4.0.5",
-    "@spectrum-css/switch": "^4.0.16",
-    "@spectrum-css/tokens": "^13.0.5",
-    "@spectrum-css/tray": "^2.1.21",
-    "gulp": "^4.0.0"
+  "peerDependenciesMeta": {
+    "@spectrum-css/checkbox": {
+      "optional": true
+    },
+    "@spectrum-css/divider": {
+      "optional": true
+    },
+    "@spectrum-css/switch": {
+      "optional": true
+    },
+    "@spectrum-css/tray": {
+      "optional": true
+    }
```

Comment:
What makes these optional and `@spectrum-css/icon` not optional?

Reply:
I think in the markup I saw it included Icon and it wasn't wrapped in an if or when so I left it off optional.  I'd defer after merge to the developer who worked on menu last to make that final call.